src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/controllers/api/UserController.kt

```diff
@@ -18,4 +23,15 @@ class UserController(context: ActionContext,
                 expiresIn = tokenHelper.issuer.tokenLifeSpan.seconds)
     }
 
+    fun addUser(): String
+    {
+        val source = UserSource.valueOf(getFromPosted("source"))
+        userRepo.addUser(getFromPosted("email"), getFromPosted("username"), getFromPosted("displayName"), source)
+        return ""
+    }
+
+    private fun getFromPosted(key: String): String
```

Comment:
This would work nicely as a method on `ActionContext` - usage `context.postData(key)`

Reply:
Good idea